simulators/portal-interop/src/main.rs

```diff
@@ -522,6 +522,24 @@ dyn_async! {
         if let Err(err) = pong {
                 panic!("Unable to receive pong info: {err:?}");
         }
+
+        // Verify that client_b stored client_a its ENR through the base layer
+        // handshake mechanism.
+        let stored_enr = match client_a.rpc.node_info().await {
+            Ok(node_info) => node_info.enr,
+            Err(err) => {
+                panic!("Error getting node info: {err:?}");
+            }
+        };
+
+        match HistoryNetworkApiClient::get_enr(&client_b.rpc, stored_enr.node_id()).await {
+            Ok(response) => {
+                if response != stored_enr {
+                    panic!("Response from GetEnr didn't return expected ENR")
```

Comment:
It's nice to throw in the values:
```suggestion
                    panic!("Response from GetEnr didn't return expected ENR. Got: {response}; Expected: {stored_enr}")
```
(or maybe it should be the debug values, like `{response:?}`)